frontend/src/lib/voter/components/navigation/VoterNav.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
```

Comment:
We don't need this component. As the navigation for the voter app or the candidate app is defined only once, we can define it on the actual route page. Also, if we need to render some nav items dynamically based on the application state it will make more sense to define the Navigation menu in the pages.

Reply:
In my opinion, it depends a bit on how the page templates that will be added later are structured, but we can discuss this related to that PR once I get it started. Even there, though, I think it might make sense that navigation was its separate file, i.e., component, because it's logically a bit different from the page template itself and would make for nicer separation in terms of possible conflicts with changes to files.

Anyway, I can leave it out from this PR.